modules/sde-common/pom.xml

```diff
@@ -2,8 +2,8 @@
 <!--
 /********************************************************************************
  * Copyright (c) 2022 BMW GmbH
- * Copyright (c) 2022, 2023 T-Systems International GmbH
- * Copyright (c) 2022, 2023 Contributors to the Eclipse Foundation
+ * Copyright (c) 2022, 2024 T-Systems International GmbH
```

Comment:
Remove the space between years => 2022,2024

Reply:
Removed